AAUnlimited/MemMods/AAPlay/Events/HInjections.cpp

```diff
@@ -67,6 +74,34 @@ bool __stdcall TickRedirect(ExtClass::HInfo* hInfo) {
 			loc_activeFaceXX = active->m_xxFace;
 			loc_passiveFaceXX = passive->m_xxFace;
 		}
+		// Detecting cumshot pose stage
+		cumshotStart = (byte*)ExtVars::ApplyRule(cumshotStartOffset);
+		if (!cumshotStarted && *cumshotStart == 1) {
+			LUA_EVENT_NORET("start_h_cumshot", hInfo);
```

Comment:
Are you going to use this for something?

Reply:
Yes, in one of the big fixes of the next PRs you see this in facecam lua.